### DISCUSSION TITLE
[solved] quasar upgrade not working

### DISCUSSION THREAD

**Discussion by jitendra1607** [quasarframework/quasar#9039] [CLI - PWA mode] [answered]
$` quasar upgrade`

  Quasar CLI · Gathering information with npm...

  Congrats! All Quasar packages are up to date.

Above command is not showing latest version i.e. 1.15.10

my package.json has "quasar": "^1.14.3"

**Comment by metalsadman** [comment, marked as answer]
You can see your local quasar version running `quasar info` within your project folder, and notice the caret `^` in your package.json it means version 1.14 and above.

**Reply by jitendra1607** [reply]
Thank you.
